.github/workflows/ui-service.yml

```
@@ -0,0 +1,75 @@
+    
+    steps:
+      - uses: actions/checkout@v4
+      
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 16
+      
+      # - name: Install dependencies
+      #   working-directory: ./
+      #   run: npm ci
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)